EIPS/eip-9999.md

```diff
@@ -0,0 +1,56 @@
+---
+eip: 9999
```

Comment:
```suggestion
eip: 7708
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

Reply:
Oh ok i don't have a clue on what to do here. I'm just trying to figure out how to borrow usd. With my collateral. Tokens.